drivers/can/Kconfig

```diff
@@ -66,6 +66,11 @@ config CAN_1
 	help
 	  Enable CAN controller 1
 
+config CAN_2
+	bool "Enable CAN 2"
+	help
```

Comment:
It's also available on F7. depends on entry would be nice here.

Reply:
F1, F2, L4 as well..
Adding suppotr for a series would require at least to provide matching fixup and ideally enabling on a board.
Otherwise I'm fine to add depends_on F4 only for now and mention potential support on other series as a comment

Reply:
@erwango updated